lib/rubocop/cop/naming/predicate_name.rb

```diff
@@ -121,6 +150,16 @@ def validate_config
 
         private
 
+        def_node_matcher :sorbet_return_type, <<~PATTERN
+          (block (send nil? :sig) args (send nil? :returns $_type))
+        PATTERN
```

Comment:
```suggestion
        def_node_matcher :sorbet_return_type, <<~PATTERN
          (block (send nil? :sig) args (send _ :returns $_type))
        PATTERN
```

Given you mentioned that `returns` can be chained on `params`, can you please make the following change, and also add a test for `sig { params(foo: T::Boolean).returns(T::Boolean) }`?